Components/login.css

```diff
@@ -0,0 +1,32 @@
+.login-container{
+    background-color: var(--color-primary);
+    width: auto;
+    height: 30rem;
+}
+.input-form{
+    width: 27rem;
+    border-radius: 1rem;
+    height: 30rem;
+    align-self: center;
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+    border-radius: 1rem;
+    color: var(--color-primary-dark);
+    background-image: url(/Assests/login-bg.png);
+}
+.input-form label, input{
+    font-size: large;
+}
+.input-form .checkbox-input-form{
+    width: auto;
+}
+.login-btn, .sign-up-btn{
+    font-weight: bold;
```

Comment:
use font-weight value instead of bold bolder

Reply:
Done, Thanks.